core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -1,9 +1,33 @@
-PLATFORM_FLAVOR ?= stm32mp157
```

Comment:
Don't you want to keep a default value? What will get built by `make PLATFORM=stm32mp1`?

Reply:
`make PLATFORM=stm32mp1` should build the base platform without any embedded DTB as prior this change.